modules/preview-react/text/lib/Title.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {Text, TypeLevelTextProps} from '@workday/canvas-kit-preview-react/text';
+
+export const Title = createComponent('h1')({
+  displayName: 'Title',
+  Component: (elemProps: TypeLevelTextProps, ref, Element) => (
+    <Text ref={ref} as={Element} level="title" {...elemProps} />
```

Comment:
I know we created `Title` cause that's part of our text tokens, but now we have a `Heading` and `Title` and they both feel very similar. Should we compromise on one or keep both?

Reply:
How does the consumer know which one to use